CSharpFundamentals/Set1-Task2/Set1-Task6/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿int[] player1Coin = new int[5] { 7, 5, 1, 25, 30 };
+int[] player2Coin = new int[5] { 5, 5, 10, 1, 25 };
+double player1TotalScore = 0;
+double player2TotalScore = 0;
+
+for (int i = 0; i < player1Coin.Length;)
+    for (int j = 0; j < player2Coin.Length;)
```

Comment:
how many times will this loop run?

Reply:
its run equal to the length of array. if array [5]. then it run 5 times. If array [6] it run 6 times.